Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer2.1.plan

```diff
@@ -1,9 +1,783 @@
 {
     "timeout": 600,
     "count": 4,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/Speedometer/@r290664",
+    "github_source": "https://github.com/WebKit/WebKit/tree/d0bce574a12b3524176d177f737f324e1694dcfc/PerformanceTests/Speedometer",
```

Comment:
Let's add `"local_git_archive":"PerformanceTests/Speedometer@d0bce574a12b3524176d177f737f324e1694dcfc",`
to optimize local running from this repo.